example/publiccode.yml

```diff
@@ -1,28 +1,34 @@
-version: 0.0.1                     # publiccode YAML standard version 0.0.1
+version: "0.1" /                    # publiccode YAML standard version 0.0.1
+url: "https://example.com/italia/repo.git"  # URL of this repository
 
 legal:
+  main-copyright-owner: Comune di Reggio Emilia e amici
+  repo-owner: Comune di Reggio Emilia # copyright owner
   authors-file: AUTHORS             # file listing copyright information
   license: AGPL-3.0-or-later        # SPDX expression of license
 
-maintainance:                        # Maintainance status as open-source software
-  - maintanance-type: commercial     # Whether there is a commercial contract of maintainance in place. No -> community
-    maintained-until: 2019-01-01     # If maintainance-type is commercial, then the maintenance contract is due
-    technical-contacts:              # Name of the people that are currently technically maintaining this software
-    - name: Francesco Rossi
-      email: francesco.rossi@comune.reggioemilia.it
-      affiliation: Comune di Reggio Emilia
-    - name: Dario Bianchi
-      email: dario.bianchi@fornitore.it
-      affiliation: Fornitore Privato Spa
+maintenance:                        # Maintainance status as open-source software
+  type: commercial                  # Whether there is a commercial contract of maintainance in place. No -> community
+  until: "2019-01-01"               # If maintainance-type is commercial, then the maintenance contract is due
+  maintainer: Company Spa           # Name of the lead maintainer
+  technical-contacts:               # Name of the people that are currently technically maintaining this software
+  - name: Francesco Rossi
+    email: "francesco.rossi@comune.reggioemilia.it"
```

Comment:
Is there a reason emails are `"` strings and the rest not?

Reply:
YAML supports both formats. I usually put quotes only when required like in this case for the special characters.

Reply:
Awesome :)